.circleci/config.yml

```diff
@@ -101,7 +101,9 @@ workflows:
             - test
           filters:
             branches:
-              only: master
+              only:
+                - master
+                - republish-all-forms
```

Comment:
~- republish-all-forms~

Reply:
Waiting to deploy to test on https://app.circleci.com/pipelines/github/ministryofjustice/fb-publisher/814/workflows/f66a7a00-98c4-40e9-ae04-429e00a887f4 so I can remove afterwards